Library/Homebrew/rubocops/cask/discontinued.rb

```diff
@@ -0,0 +1,43 @@
+# typed: true
+# frozen_string_literal: true
+
+# TODO: Remove this when we remove `caveats { discontinued }` (likely for Homebrew 4.5.0)
```

Comment:
It may be worth keeping beyond that just because it's nice to be able to warn people about this rather than the slightly harsh error messages they will get on removal.

Reply:
Yeah we've kept a number of old cops like that - you can see some by grepping `deprecated` in `Library/Homebrew/rubocops/`.